cypress/integration/member-membership.spec.js

```diff
@@ -0,0 +1,114 @@
+import faker from 'faker'
+
+describe('Member Membership', () => {
+  beforeEach(() => {
+    cy.request('/cypress_rails_reset_state')
+  })
+
+  it('renders the membership page', () => {
```

Comment:
Navigate to certain page is something you can easily achieve with `react-testing-library` mounting the application as it suggested by the library and just click to get there. On top of that, I wouldn't even bother to have a test of a link that goes where I want.

Nevertheless, all the above is a broader conversation. My point to you is **for a future project** try to conceptualise what should be on a cypress run vs what should be within integration testing using something like `react-testing-library`.

It is far more cheap to run a jest test suite than a cypress run.

Reply:
yeah that could be right, but this also tests the view rendering stack since we are not just testing we are rendering a component, which is where `react-testing-library` shines.